.github/workflows/tests.yaml

```diff
@@ -40,35 +40,36 @@ jobs:
         env:
           CI: true
 
-  cli-package-test:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v3
-      - name: Cache dependencies
-        uses: actions/cache@v3
-        with:
-          path: ~/.npm
-          key: npm-${{ hashFiles('package-lock.json') }}
-          restore-keys: |
-            npm-${{ hashFiles('package-lock.json') }}
-            npm-
-      - name: Environments
-        run: |
-          echo
-          echo Using Node $(node -v), NPM $(npm -v)
-          echo
-      - name: Install dependencies
-        run: npm ci
-        env:
-          CI: true
-      - name: Prepare CLI package
-        run: npm run pack:prepare
-      - name: Install CLI Package
-        run: npm install -g redocly-cli.tgz
-      - name: Redocly version
-        run: redocly --version
-      - name: Definition test
-        run: redocly lint packages/core/src/benchmark/benches/rebilly.yaml --format=stylish
+  # FIXME: fix this so the `cli` package includes the corresponding `core`
```

Comment:
Should this be addressed when?

Reply:
I reopened the [original issue](https://github.com/Redocly/redocly-cli/issues/732).